app/ethermint.go

```diff
@@ -36,13 +36,14 @@ import (
 )
 
 const appName = "Ethermint"
+const shortAppName = "emint"
 
 var (
 	// DefaultCLIHome sets the default home directories for the application CLI
-	DefaultCLIHome = os.ExpandEnv("$HOME/.emintcli")
+	DefaultCLIHome = os.ExpandEnv("$HOME/." + shortAppName + "cli")
 
 	// DefaultNodeHome sets the folder where the applcation data and configuration will be stored
-	DefaultNodeHome = os.ExpandEnv("$HOME/.emintd")
+	DefaultNodeHome = os.ExpandEnv("$HOME/." + shortAppName + "d")
```

Comment:
these aren't being threaded through. These are still emint{d, cli} when overriding the name. Test this by setting the app name and when you run the daemon init, make sure that it stores data to this updated directory